packages/components/.storybook/config.tsx

```diff
@@ -4,43 +4,39 @@ import { withKnobs } from '@storybook/addon-knobs';
 import { withA11y } from '@storybook/addon-a11y';
 import { addDecorator, addParameters, configure } from '@storybook/react';
 import { themes } from '@storybook/theming';
-import { ThemeDecorator } from '../src/stories/decorators';
+import { ThemeDecorator } from './decorators';
 import { createGlobalStyle } from 'styled-components';
 import global from '@codesandbox/common/lib/global.css';
+import theme from './theme';
 
 const viewports = {
   mobile: {
     name: 'Mobile',
-    styles: {
-      width: '375px',
-      height: '667px',
-    },
+    styles: { width: '375px', height: '667px' },
   },
   tablet: {
     name: 'Tablet',
-    styles: {
-      width: '768px',
-      height: '1024px',
-    },
+    styles: { width: '768px', height: '1024px' },
   },
   laptop: {
     name: 'Laptop',
-    styles: {
-      width: '1366px',
-      height: '768px',
-    },
+    styles: { width: '1366px', height: '768px' },
   },
   desktop: {
     name: 'Desktop',
-    styles: {
-      width: '1920px',
-      height: '1080px',
-    },
+    styles: { width: '1920px', height: '1080px' },
   },
 };
 
+// new globals based on theme?
+// using sidebar as the styles for body for now 🤷
 const GlobalStyle = createGlobalStyle`
-  ${global}
+  ${global};
+  body {
+    font-family: 'Inter', sans-serif;
+    background-color: ${theme.colors.sideBar.background};
+    color: ${theme.colors.sideBar.foreground};
```

Comment:
@SaraVieira Added some globals here to help in development, not sure if this is the right place though

Reply:
Seems like it! Thank you!